doto-backend/index.js

```diff
@@ -1,42 +1,40 @@
 const express = require('express')
 const cors = require('cors')
 const app = express()
-const apiPort = process.env.PORT || 3000
+const apiPort = 3000
 require('dotenv').config();
 
 // Mongoose connection
 const mongoose = require('mongoose');
-// db connection string will point to Azure string only in production, fallbacks to dev database string 
-const connectionString = process.env.AZURE_CONN || process.env.mongodb_uri;
-// Add authentication strings only in production environment
-const connParams = { useNewUrlParser: true }
-if(process.env.AZURE_USER && process.env.AZURE_PW){
-    connParams.auth = {
-        user : process.env.AZURE_USER,
-        password: process.env.AZURE_PW
-    };
-}
-mongoose.connect(connectionString, connParams);
+mongoose.connect(process.env.mongodb_uri, { useNewUrlParser: true });
 const db = mongoose.connection;
 
 // Checking for DB connection
 db.once('open', function(){
     console.log("Connected to MongoDB.");
 });
 db.on('error', function(){
-    console.log(err);
+    
```

Comment:
Hi, just wondering if there a reason for removing this line? If the line is not needed, you could probably remove the entire function.

Reply:
It was having compatibility issues with some people's environments. It's probably better to remove it @fsan110

Reply:
Ah ok